app/includes/header.njk

```diff
@@ -19,14 +19,17 @@
   }) }}
 </span>
 
+{% if loginEnabled === 'true' %}
+  {% set innerComponent = loginLogout({
+      dataTestId: "login-logout-component",
+      username: username,
+      loginUrl: "/login",
+      logoutUrl: "/logout"
+    }) %}
+{% endif %}
+
 {{ bcHeader({
   dataTestId: "header-banner",
-  innerComponent: loginLogout({
-    dataTestId: "login-logout-component",
-    username: username,
-    loginUrl: "/login",
-    logoutUrl: "/logout"
-  }),
+  innerComponent: innerComponent,
```

Comment:
when login is disabled this innerComponent will be undefined? does nunjucks not complain about this

Reply:
no it doesn't care because it is checked in the bcheader component if it is undefined